### PR TITLE
2018 summer events

### PR DESCRIPTION
**Pull request by dasanchez**
Added the following dates:
May 17: Planning session @ HackLab.TO
June 21: Install party @ Free Geek
July 26: Install party @ HackLab.TO
August 23: Install party @ Free Geek

### CODE REVIEW

**Review by darkdrgn2k — APPROVED** [review]

File: _posts/2018-06-21-install-party.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+date: 2018-06-21
+startTime: '19:00'
+endTime: '21:00'
+---
+
+Join us at {{page.startTime | date:"%-I:%M%P "}} on {{ page.date | date: '%B %d, %Y' }} for our monthly install party. Come get your mesh node set up and play with distributed applications! This month we will be meeting at [{{ page.location }}]({{ page.locationLink }}) for a hands-on session.
+
+Do you have any e-waste you would like to donate to Free Geek? Bring it! See their list of [accepted electronics here](https://www.freegeektoronto.org/pick-up/).
+
+Planning on coming? Let us know through our [chat channel](https://chat.tomesh.net/#/room/#tomesh:tomesh.net) or email us at <hello@tomesh.net>.
```
Comment (dcwalk) [inline _posts/2018-06-21-install-party.md]: Use the {{ site.email }} variable?

File: _posts/2018-07-26-install-party.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+location: HackLab.TO, 1266 Queen St
+locationLink: https://www.openstreetmap.org/node/3522641496
+date: 2018-07-26
+startTime: '19:00'
+endTime: '21:00'
+---
+
+Join us at {{page.startTime | date:"%-I:%M%P "}} on {{ page.date | date: '%B %d, %Y' }} for our monthly install party. Come get your mesh node set up and play with distributed applications! This month we will be meeting at [{{ page.location }}]({{ page.locationLink }}) for a hands-on session.
+
+Planning on coming? Let us know through our [chat channel](https://chat.tomesh.net/#/room/#tomesh:tomesh.net) or email us at <hello@tomesh.net>.
```
Comment (dcwalk) [inline _posts/2018-07-26-install-party.md]: Use the {{ site.email }} variable?

File: _posts/2018-08-23-install-party.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+date: 2018-08-23
+startTime: '19:00'
+endTime: '21:00'
+---
+
+Join us at {{page.startTime | date:"%-I:%M%P "}} on {{ page.date | date: '%B %d, %Y' }} for our monthly install party. Come get your mesh node set up and play with distributed applications! This month we will be meeting at [{{ page.location }}]({{ page.locationLink }}) for a hands-on session.
+
+Do you have any e-waste you would like to donate to Free Geek? Bring it! See their list of [accepted electronics here](https://www.freegeektoronto.org/pick-up/).
+
+Planning on coming? Let us know through our [chat channel](https://chat.tomesh.net/#/room/#tomesh:tomesh.net) or email us at <hello@tomesh.net>.
```
Comment (dcwalk) [inline _posts/2018-08-23-install-party.md]: Use the {{ site.email }} variable?

**Review by dcwalk — COMMENTED** [review]
Just a thought -- July and August tend to be pretty quiet months.

I'm just wondering whether July (which will have potentially 1. Maker Fest, 2. Our Networks, 3. People at HOPE over the first three weekends) might be okay to skip, or reconfigure into like just a social event?

Meet and greet for new to mesh people that can be mentioned at all those events?

**Comment by darkdrgn2k** [comment]
Maker fest deadline is this Monday and as far as I know no one has submitted yet so I don't know if we will be there... 

Just a though but maybe doing a install night as part of state of our networks...?

**Comment by dcwalk** [comment]
@darkdrgn2k -- I don't think an install _night_ will fit within the current schedule for our networks Fri-Sun. I think during the sprints (Mon-Wed) there was early discussion of tomesh hacking, which would basically do the same thing

**Comment by dasanchez** [comment]
@dcwalk I'm OK with making the July meetup at HackLab a Meet and Greet. Or would you prefer skipping HackLab altogether to meet at the Rhino or elsewhere instead?

**Comment by dcwalk** [comment]
@dasanchez -- I think however you want to proceed is fine! (Sorry for relentless proposing!)

File: _posts/2018-05-17-planning-session.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+date: 2018-05-17
+startTime: '18:30'
+endTime: '21:30'
+---
+
+Join us at our sixth planning session, where we regroup and plan for the summer! During this session we set priorities and projects, and form **working groups** for the next six months and identify those interested in leading projects. You can see our [current working groups and governance processes](https://github.com/tomeshnet/documents/blob/master/governance/coordination-structure.md) to get a sense of existing interests!
+
+We will break off to discuss ongoing deployment efforts afterwards.
+
+We will meet at {{page.startTime | date:"%-I:%M %P "}} in [{{ page.location }}]({{ page.locationLink }}).
```
Comment (garrying) [inline _posts/2018-05-17-planning-session.md]: 2 trailing spaces at the line of this line.

File: _posts/2018-06-21-install-party.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: "Install Party"
+slug: install-party
+text: 'Bring your hardware to set up a mesh node'
+location: Free Geek Toronto, 180 Sudbury St
```
Comment (garrying) [inline _posts/2018-06-21-install-party.md]: 2 trailing spaces at the line of this line.

File: _posts/2018-08-23-install-party.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: "Install Party"
+slug: install-party
+text: 'Bring your hardware to set up a mesh node'
+location: Free Geek Toronto, 180 Sudbury St
```
Comment (garrying) [inline _posts/2018-08-23-install-party.md]: 2 trailing spaces at the line of this line.

**Comment by dasanchez** [comment]
Thanks @garrying ! I removed those trailing spaces.
@dcwalk never relent 😉

**Comment by Shrinks99** [comment]
Changed the message for the planning session, this should be more applicable to what we are discussing this time around (and also what I hope to accomplish at this one ;P )

**Pull request merged**